ddtrace/tracer/option_test.go

```diff
@@ -76,13 +78,32 @@ func TestTracerOptionsDefaults(t *testing.T) {
 			assert.Equal(t, c.dogstatsdAddr, "my-host:123")
 		})
 
+		t.Run("env-env", func(t *testing.T) {
+			os.Setenv("DD_ENV", "testEnv")
+			defer os.Unsetenv("DD_ENV")
+			tracer := newTracer()
+			c := tracer.config
+			assert.Equal(t, "testEnv", c.globalTags[ext.Environment])
+		})
+
 		t.Run("option", func(t *testing.T) {
 			tracer := newTracer(WithDogstatsdAddress("10.1.0.12:4002"))
 			c := tracer.config
 			assert.Equal(t, c.dogstatsdAddr, "10.1.0.12:4002")
 		})
 	})
 
+	t.Run("envOverride", func(t *testing.T) {
```

Comment:
How about `override`? We don't have any others anyway. We don't use `camelCase` naming anywhere.

Reply:
👍